website/en/docs/types/functions.md

```diff
@@ -321,6 +321,10 @@ add.bar = "hello world";
 
 ### `Function` Type <a class="toc" id="toc-function-type" href="#toc-function-type"></a>
 
+** NOTE: `Function` has become an alias to `any`. For new code, either use `any` or
```

Comment:
I don't feel very strongly about this, but you could tone it down a bit:
"For new code prefer `any` or `(...args: Array<any>) => any`. `Function` will be deprecated and removed in a future version of Flow."